docs/Documentation/ForcingData.md

```diff
@@ -37,39 +39,7 @@ As of release 4.1.2, we have fixed inconsistencies (present in all previous rele
-| PREC              | total precipitation                                                                               | mm per step       | mm/s == kg/m2/s   |
-| PRESSURE          | atmospheric pressure                                                                              | kPa               | Pa                |
-| QAIR              | specific humidity                                                                                 | kg/kg             | kg/kg             |
-| RAINF             | total rainfall                                                                                    | mm per step       | mm/s == kg/m2/s   |
-| REL_HUMID         | relative humidity                                                                                 | fraction          | fraction          |
-| SHORTWAVE         | incoming shortwave (solar) radiation                                                              | W/m2              | W/m2              |
-| SNOWF             | total snowfall                                                                                    | mm per step       | mm/s == kg/m2/s   |
-| TMAX              | daily maximum temperature                                                                         | deg C             | K                 |
-| TMIN              | daily minimum temperature                                                                         | deg C             | K                 |
-| TSKC              | cloud cover                                                                                       | fraction          | fraction          |
-| VEGCOVER          | partial veg cover fraction                                                                        | fraction          | fraction          |
-| VP                | atmospheric vapor pressure                                                                        | kPa               | Pa                |
-| WIND              | wind speed                                                                                        | m/s               | m/s               |
-| WIND_E            | East component of wind speed                                                                      | m/s               | m/s               |
-| WIND_N            | North component of wind speed                                                                     | m/s               | m/s               |
-| SKIP              | indicates a data column which is ignored, <br>e.g., year, month, and day columns if they are present  |               |                   |
-
-By default, the variables are assumed to have units desribed in column 3 of the table above. To use the units in column 4 (ALMA), the user must set ALMA_INPUT TRUE in the [global parameter file](GlobalParam.md). ALMA units correspond more closely to the units used by reanalysis products or GCMs.
```

Comment:
Note, I removed these lines because they are redundant with the table in InputVarList.md (which there already is a link to in this file).  Just reducing the chances that the two tables get out of sync with each other (and reducing potential confusion from the reader as to where the table is).